Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3440,54 +3428,46 @@ - (void)sendMessageAttachment:(SignalAttachment *)attachment
 
 - (NSURL *)videoTempFolder
 {
-    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
-    NSString *basePath = ([paths count] > 0) ? [paths objectAtIndex:0] : nil;
-    basePath = [basePath stringByAppendingPathComponent:@"videos"];
-    if (![[NSFileManager defaultManager] fileExistsAtPath:basePath]) {
-        [[NSFileManager defaultManager] createDirectoryAtPath:basePath
+    NSString *temporaryDirectory = NSTemporaryDirectory();
```

Comment:
A small but key change: by creating video temp files in the "temp" folder rather than the "cache" folder, we cue iOS to clean them up sooner.